src/content/_data/events.yaml

```diff
@@ -122,25 +121,26 @@ mind-matters:
     - Multiple-choice questions based on Computer Science and IT world.
     - The qualifying teams would compete in the Event's Main Quiz.
 
-spin-a-web:
-  title: Spin a Web
+coding-cup:
+  title: DUCS Coding Cup
+  quote:
+    text:If debugging is the process of removing software bugs, then programming must be the process of putting them in. 
+    by:Edsger Dijkstra
   description:  |-
-    Have you been bitten by the creative bug? Then we have just the right antidote.
-    Spin on the creativity loom a web-site of Imagination, Innovation and Design.
+    This is a special event mainly targeting the students of undergraduate pursuing their bachelor’s in Computer Science from Delhi University. 
+    Students will gain ample amount of knowledge and practice which will help them increase their coding skills. 
   islive: false
   isover: false
   isnontech: false
   image: https://2.bp.blogspot.com/-8R7BMj3aL5g/WoVRk4PjVlI/AAAAAAAAAHw/1qje7sRyf1EAw73fgaP7kmC2wm1yGhwJwCLcBGAs/s1600/spin-a-web.png
   teamSize: 2
-  lang: js
+  lang: c/c++/java/python
```

Comment:
this `lang` can only be a single value. pick either of cpp/python/js which ever suits most